workbench-core/repo-scripts/github-oidc/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@aws/workbench-core-github-oidc",
```

Comment:
Should this be a package? in the SWB repo? -- we can sync offline -- curious to know what SWB SDEs think

Reply:
We are not going to publish it. but are you thinking from security point of view ?